src/common/__tests__/useWindowSize-test.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { act } from 'react-dom/test-utils';
+import { mount, ReactWrapper } from 'enzyme';
+import useWindowSize from '../useWindowSize';
+
+describe('useWindowSize', () => {
+    function TestComponent(): JSX.Element {
+        const windowSize = useWindowSize();
+
+        return (
+            <div id="container">
+                <span id="height">{windowSize?.height}</span>
```

Comment:
Is this the way in TypeScript to check for existence of windowSize before applying the `height` property?

Reply:
Optional chaining https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining